tools/pubsys/src/repo/validate_repo/mod.rs

```diff
@@ -164,8 +173,12 @@ mod error {
         #[snafu(display("Missing target: {}", target))]
         TargetMissing { target: String },
 
+        #[snafu(display("Unable to create thread pool: {}", source))]
+        ThreadPool { source: rayon::ThreadPoolBuildError },
+
         #[snafu(display("Failed to join thread: {}", inner))]
         Join { inner: String },
     }
 }
 pub(crate) use error::Error;
+use std::cmp::min;
```

Comment:
nit: can this go at the top with the other normal use statements?  The Error use is only here because it's conceptually tied to the error module.

Reply:
It's my IDE again. I have to say, I'm not a fan of use statements that aren't at the top. Seems like my IDE agrees with me. :)